src/styles/components/_button.scss

```
@@ -1,18 +1,128 @@
 .button {
   cursor: pointer;
-  height: 40px;
-  width: 100%;
-  background-color: var(--gray-1);
+  border: none;
+  background-color: var(--white);
   font: var(--button);
+}
+
+.button--large {
+  height: 40px;
+  width: 295px;
   color: var(--gray-6);
+  background-color: var(--gray-1);
+  border-radius: var(--border-radius);
+  border: 1px solid var(--gray-1);
+}
+
+.button--squareSharp {
+  height: 22px;
+  width: 22px;
+  color: var(--gray-1);
+  background-color: var(--white);
   border-radius: var(--border-radius);
   border: 1px solid var(--gray-1);
+  padding: 0em;
+}
+
+.button--squareSharp:hover {
+  background-color: var(--gray-1);
+}
+
+.button--squareSharp:hover path {
+  stroke: #f2f2f2;
 }
 
 .button--floating {
   box-shadow: 0px 0px 20px rgba(0, 0, 0, 0.5);
 }
 
+.button--qtd {
+  width: 28px;
+  height: 18px;
+  /* Red */
+  background: var(--red);
+  border-radius: 18px;
+}
+
+.button--qtd-small {
+  width: 18px;
+}
+
+.button--qtdinfo {
+  left: 15.48%;
+  right: 9.52%;
+  top: 16.67%;
+  bottom: 16.67%;
+  padding: 0px;
+  font-family: Noto Sans;
```

Tem um erro de linter aqui
```suggestion
  font-family: "Noto Sans";
```